AcquisitionInfo/explainer.md

```diff
@@ -0,0 +1,296 @@
+```
+
+`installSource` is a required dictionary property with values dependent on the following scenarios:
+
+1. `Microsoft Store` | `Google Play Store` | `Apple App Store` - The web app was installed through one of these store platforms.
+
+2. `Policy` - The web app was installed due to an enterprise policy.
+
+3. `<install origin>` - For [same-domain installation](#same-domain-installation), the domain name of the web app.
+For [cross-domain installation](#web-install-api-cross-domain-installation), the installer's domain name.
+
+While `installSource` remains a required property, depending on the platform of the acquisition the pairs contained within
+the dictionary may differ to fit different store use cases.
+
+> Payload examples are taken from [Play Install Referrer][Play Install Referrer API] and [Apple Ads Attribution][Apple Ads Attribution API]
+documentation and are not final.
+
+```js
+details = {
+  installSource: "Google Play Store",
+  installReferrer: "utm_source=google-play&utm_medium=organic",
+  referrerClickTimestampSeconds: 1623214800,
+  installBeginTimestampSeconds: 1623214810,
+  installBeginReferrer: "utm_source=google-play&utm_medium=referral&utm_campaign=campaign-123",
+  installVersion: "1.0.0",
+  googlePlayInstant: false
+}
+```
+
+```js
+details = {
+  installSource: "Apple App Store",
+  attribution: "true",
+  orgId: "40669820",
+  campaignId: "542370539",
+  conversionType: "Download",
+  adGroupId: "542317095",
+  countryOrRegion: "US",
+  keywordId: "87675432",
+  adId: "542317136"
+}
+```
+
+In the event that a web application has referral information that has expired for that specific
+application, or in any other case where acquisition info is missing for the web application,
+the API would return empty values for the associated properties in the dictionary. This is consistent with
+existing behavior in store information retrieval API for native applications.
+
+```js
+details = {
+  installSource: "Microsoft Store",
+  campaignId: ""
+}
+```
+
+## Attribution on browser installs
+
+While attribution is currently not being tracked in the same capacity as Store installations for browser-initiated
+installations of web applications, the Acquisition Info API could enable this sort of attribution capability.
+
+### Same-domain installation
+
+Currently the main method of installing a PWA through the browser is navigating to the desired web application and
+initiating a same-domain installation. It may be the case that the intial navigation was due to an ad click.
+By having a standardized dictionary property (`attributionId`) that the UA is responsible for capturing to keep track of ad attribution,
+it becomes possible for the Acquistion Info API to return attribution information for browser-installed
+web applications as well. This `attributionId` dictionary property could be captured by the UA through a query parameter such
+as `__a_id` in the [HTTP header `Referer`][HTTP Referer] field.
```

Comment:
This doesn't sound quite right.  The `HTTP Referer` would give you the URL that the user was _previously_ on.  In this scenario, the `__a_id` GET parameter would be in the URL for the page that the user is _currently_ on.  Therefore, this would be unaffected by the UA's referer policy.